Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml

```diff
@@ -13,25 +13,28 @@
   - type: SolutionContainerManager
     solutions:
       food:
-        maxVol: 20
+        maxVol: 36
         reagents:
         - ReagentId: Nutriment
-          Quantity: 15
+          Quantity: 30
   - type: SliceableFood
-    count: 8
+    count: 6
   - type: Item
     size: 8
   - type: Tag
     tags:
     - Pizza
 
 - type: entity
-  parent: FoodPizzaBase
+  parent: FoodInjectableBase # Not sliceable
```

Comment:
needs retagged as pizza then

Reply:
wdym? It is tagged
![image](https://user-images.githubusercontent.com/36813380/143673796-49945a5c-8889-4009-901b-dc4b014f9987.png)